### PR TITLE
Blank Workout can't be passed into DB

### PR DESCRIPTION
**Pull request by josephferrairo**
I modified the workout model so that the user cannot input a blank workout, I also modified the workout controller so that it gives the user an error when the user passes a blank workout. I also added tests for new and create workout as well as added a line to the Database cleaner file so that it cleans the db after each test is run. I also removed the asterisk next to required fields in forms.

### CODE REVIEW

File: config/locales/simple_form.en.yml [hunk 1]
```diff
@@ -4,7 +4,7 @@ en:
     "no": 'No'
     required:
       text: 'required'
-      mark: '*'
```
Comment (BGRicker) [inline config/locales/simple_form.en.yml]: huh?

File: spec/controllers/workouts_controller_spec.rb [hunk 2]
```diff
@@ -9,4 +9,20 @@
...
+  end
+
+  describe "POST #create" do
+    login_user
+    context "a successful create" do
+      it "saves the customer object" do
+        expect { post :create, params: { workout: Fabricate.attributes_for(:workout) } }.to change( Workout, :count).by(1)
+      end
+    end
+  end
```
Comment (BGRicker) [inline spec/controllers/workouts_controller_spec.rb]: do we already have a spec for a workout that's declined for no title? the happy specs are nice but we need sad ones too

File: spec/support/database_cleaner.rb [hunk 3]
```diff
@@ -18,4 +18,8 @@
 	config.after(:each) do
 		DatabaseCleaner.start
 	end
+
+	config.after(:each) do
+		DatabaseCleaner.clean
```
Comment (BGRicker) [inline spec/support/database_cleaner.rb]: a second config.after(:each) loop probably isn't necessary

File: config/locales/simple_form.en.yml (hunk 1 above)
Comment (BGRicker) [inline config/locales/simple_form.en.yml]: ahhh read the description 👍

File: spec/support/database_cleaner.rb (hunk 3 above)
Comment (josephferrairo) [inline spec/support/database_cleaner.rb]: I was going by the format of the file

Comment (josephferrairo) [inline spec/support/database_cleaner.rb]: each instruction has its own :each loop

Comment (josephferrairo) [inline spec/support/database_cleaner.rb]: `RSpec.configure do |config|
	config.before(:suite) do
		DatabaseCleaner.clean_with(:truncation)
	end

	config.before(:each) do
		DatabaseCleaner.strategy = :transaction
	end

	config.before(:each, js: true) do
		DatabaseCleaner.strategy = :truncation
	end`

Comment (josephferrairo) [inline spec/support/database_cleaner.rb]: http://tomdallimore.com/blog/taking-the-test-trash-out-with-databasecleaner-and-rspec/

File: spec/controllers/workouts_controller_spec.rb (hunk 2 above)
Comment (josephferrairo) [inline spec/controllers/workouts_controller_spec.rb]: Added a sad spec for workouts

**Comment by josephferrairo** [comment]
Is this ok to merge or would you like for me to make some more adjustments?

**Pull request merged**